resources/Assets.md

```diff
@@ -2,13 +2,46 @@
 
 ## Assets Description
 
+Assuming the following supply chain:
+```
+MSD -> DHL -> KLM -> DHL -> Recipient
```

Comment:
I think we should stick to the flow already discussed for sake of simplicity:
MSD -> DHL -> KLM -> End recipient

Unless there is something critical we need to show adding a new interaction